collector/lib/CollectorConfig.h

```diff
@@ -113,6 +114,12 @@ class CollectorConfig {
   // Size of one ring buffer, in bytes
   unsigned int sinsp_buffer_size_;
 
+  // Max size of the thread cache. This parameter essentially translated into
+  // the upper boundary for memory consumption. Note that Falco puts it's own
+  // upper limit on top of this value, m_thread_table_absolute_max_size, which
+  // is 2^17 (131072).
+  unsigned int sinsp_thread_cache_size_ = 524288;
```

Comment:
Should we just go ahead and make this the max falco value since the current setting will be altered by falco?

Reply:
Yeah, let me do this. As discussed, I'm also going to make the thread cache twice as smaller.